MANIFESTO.md

```diff
@@ -1 +1,6 @@
 
+##Community
+
+Mankind did not evolve by sitting alone in the dark. For the software industry to really grow in the Caribbean we need collaboration, knowledge and skill sharing. Avoid platforms and tools that do not respect communities that form around them or worse; shun them.
+
+Community does not only mean software developers, after all we need to solve somebody's problems. Developers should therefore take time to interact with and integrate with the rest of society.
```

Comment:
Not all software developers are attempting to solve someone else's problems so I think this critique is too specific and therefore inappropriate. There are many different types of developers engaged in a variety of jobs or interests and we should speak to the issues that face all or most of them. This also assumes that Caribbean developers are isolated from the rest of society; is that a fair point to make, what evidence do we have of that?